draft-ietf-cose-merkle-tree-proofs.md

```diff
@@ -163,8 +163,8 @@ with the following initial contents:
 | Identifier  | Proof Type   | Proof Value | Reference
 |---
 |0            | N/A          | N/A | N/A
-|TBD_2        | inclusion    | bstr or list of bstr | {{sec-generic-inclusion-proof}}
```

Comment:
Technically, that would be:

```
bstr / [ 2* bstr ]
```
or
```
one-or-more<T> = T / [ 2* T ]
one-or-more<bstr> 
```
and not
```
[ + bstr ]
```

as illustrated in https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/issues/109#issue-1935673877